src/config/constants.ts

```diff
@@ -601,6 +603,11 @@ export const CLUSTER_COMMAND = {
     }
 }
 
+export const KIND = {
+    INGRESS: 'Ingress',
+    SERVICE: 'Service'
+}
```

Comment:
Can you change it to an enum instead of a const so that it can be used as both type & const val?

Reply:
done